app/models/user.rb

```diff
@@ -1,12 +1,30 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly make sure `:confirmable` is enabled in your user model, users will be asked to confirm  email on signup 👍🏿

Reply:
@prolajumokeoni !
Thank you for reviewing my code. I will change it.